cis/system/vm-hsp-nist-cm-11-1-allow-user-installation-of-software-only-with-explicit-privileged-status.yaml

```diff
@@ -0,0 +1,24 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorHostPolicy
+metadata:
+  name: vm-hsp-nist-cm-11-1-allow-user-installation-of-software-only-with-explicit-privileged-status
```

Comment:
The tag says this is `CM11-(2)` but the policy name suggests this as `cm-11-1`
please verify @AbhinavCSY